docs/security/02_audits.md

```
@@ -33,3 +33,4 @@
 | 2025-09     | <span id="convertible-deposits"></span>Convertible Deposits                                                         | Guardian Audits                | Convertible Deposits                         | [Link](https://storage.googleapis.com/olympusdao-landing-page-reports/audits/2025-09_Convertible_Deposits-Guardian.pdf)     |
 | 2025-10     | Convertible Deposits                                                                                                | Trust                          | Convertible Deposits                         | [Link](https://storage.googleapis.com/olympusdao-landing-page-reports/audits/2025-10_Convertible_Deposits-Trust.pdf)        |
 | 2026-02     | <span id="ohm-v1-migrator"></span>OHM v1 Migrator                                                                   | Guardian                       | OHM v1 migration and TokenMigrator defunding | [Link](https://storage.googleapis.com/olympusdao-landing-page-reports/audits/2026-02_Migrator.pdf)                          |
+| 2026-06     | <span id="layerzero-bridge"></span>LayerZero Bridge                                                                 | Guardian                       | LayerZero Bridge Security Upgrade            | [Link](https://storage.googleapis.com/olympusdao-landing-page-reports/audits/2026-06-Bridge.pdf)                            |
```

markdownlint-cli2 (0.22.1)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)